app/views/devices_guidance/how_to_order.html.erb

```diff
@@ -325,7 +325,7 @@
 
           <div class="app-step-nav__panel" id="step-panel-receive-and-distribute-laptops-and-tablets-6">
             <p class="app-step-nav__paragraph">
-              Most devices will be delivered to schools within 2 working days of an order being confirmed, subject to stock availability. For large orders (more than 150 devices), we’ll call the main delivery contact to arrange delivery.
+              <%= render partial: 'shared/delivery_message' %> For large orders (more than 150 devices), we’ll call the main delivery contact to arrange delivery.
```

Comment:
If this is a singular string, this might be better placed in the locales file (`config/locales/en.yml`) and referenced with `<%= t('key.name') %>`